packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -1487,6 +1489,33 @@ export interface DiagnosticsCallFeatureState {
     network: NetworkDiagnosticsState;
 }
 
+// @public
+export const Dialpad: (props: DialpadProps) => JSX.Element;
```

Comment:
Presume the stable api file needs recompiled but ensure this file is not included in the final PR

Reply:
Oh I guess it's because it's marked as beta, but not compiled out of stable.. given it's not part of the communication-react api file I'm OK with not beta compiling it out as long as it's not being released in the npm package stable file

Reply:
We should stable-compile this feature out of stable builds, or make the feature `_Internal`